ReactNativeClient/lib/models/Setting.js

```diff
@@ -252,7 +252,13 @@ class Setting extends BaseModel {
 					return options;
 				},
 			},
-
+			spellcheck: {
+				value: true,
+				type: Setting.TYPE_BOOL,
+				public: true,
+				appTypes: ['desktop'],
+				label: () => _('Enable Spellcheck (BETA: Only works with CodeMirror)'),
```

Comment:
We don't translate beta text, but when if we release this it probably won't be as beta, so just `_('Enable spellcheck`) (no uppercase on "spellcheck")